### PR TITLE
[RR-146] Migrate/Import issued to not ready cluster crashes

### PR DESCRIPTION
**Pull request by sjpotter**
- add a test that duplicates bug as is
- prevent migrate command and import from attempting to run on a not ready cluster

### CODE REVIEW

File: src/migrate.c [hunk 1]
```diff
@@ -140,6 +140,10 @@ int cmdRaftImport(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
 {
     RedisRaftCtx *rr = &redis_raft;
 
+    if (checkRaftState(rr, ctx) == RR_ERROR) {
```
Comment (tezc) [inline src/migrate.c:143]: Do you think we should also check if we are leader here? Something like:

```c
    if (!raft_is_leader(rr->raft)) {
        Node *leader = getLeaderNodeOrReply(rr, ctx);
        if (leader) {
            redirectCommand(rr, ctx, leader);
        }
        return;
    }
```

**Review by tezc — APPROVED** [review]
